configure.ac

```diff
@@ -1,8 +1,10 @@
 dnl required version of autoconf
 AC_PREREQ([2.53])
 
-dnl TODO: fill in your package name and package version here
-AC_INIT([gst-perf],[0.2.4])
+dnl package name and package version
+dnl NOTE: whenever you change this, also update debian/changelog
+dnl (e.g., run `dch -i`).
+AC_INIT([gst-perf],[0.2.5])
```

Comment:
We follow a home-grown version [semanthic versioning](https://semver.org), so given `major.minor.micro` we reserve the `micro` for critical bug fixes. This would be a `minor` change, but we don't increase them with every new feature. We typically group a bunch of new features and then do a release where we bump the version.

Having said that, I think that this version change will be done by the engineer making the release (probably me) in a "release" branch.

Reply:
I do like the comment though, please keep it.